packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/communication-prefs.js

```diff
@@ -10,22 +10,24 @@
 
 const BaseGroupingRule = require('./base');
 
-const AVAILABLE_LANGUAGES = [
-  'de',
-  'en',
-  'en-[a-z]{2}',
-  'es',
-  'es-[a-z]{2}',
-  'fr',
-  'hu',
-  'id',
-  'pl',
-  'pt-br',
-  'ru',
-  'zh-tw',
```

Comment:
Same here, assuming `zh` should apply to `zh-cn` is probably *not* the right thing to do as the `zh-tw` means Taiwan only, and `zh-cn` means China. The politics of sending newsletters tailored to Taiwan to Chinese users could be politically messy. Plus @l-hedgehog runs a separate Chinese stack where newsletters are specifically disabled. See https://github.com/mozilla/fxa/blob/7228646ae64fb7ddce9071e567c94b1041382bcd/packages/fxa-content-server/server/lib/configuration.js#L367 and https://github.com/mozilla/fxa/commit/5c42ab2b2a6012d60ff77e56ab7d69df52f2b7d4.

Reply:
Yup, the new code will preserve the distinction and strictly match on `zh-tw`.